zapisy/apps/notifications/signals.py

```diff
@@ -177,6 +177,25 @@ def notify_board_members_about_voting(sender: Thesis, **kwargs) -> None:
         }, target))
 
 
+@receiver(event_decision, sender=None)
+def notify_event_creator_about_decision(sender: None, **kwargs) -> None:
+    event = kwargs['event']
+    target = reverse('events:show', args=[event.id])
+    status = ""
+    if event.status == event.STATUS_ACCEPTED:
+        status = "zaakceptowane"
+    else:
+        status = "odrzucone"
```

Comment:
```suggestion
    if event.status == event.STATUS_ACCEPTED:
        status = "zaakceptowane"
    else:
        status = "odrzucone"
```
Ja to w ogóle zrobiłbym jedno przypisanie z operatorem warunkowym, ale może to tak naprawdę nie upiększa kodu?